core/internal/cltest/cltest.go

```diff
@@ -927,7 +927,7 @@ func WaitForPipeline(t testing.TB, nodeID int, jobID int32, expectedPipelineRuns
 
 		var matched []pipeline.Run
 		for _, pr := range prs {
-			if pr.State != state {
+			if !pr.State.Finished() && pr.State != state {
```

Comment:
Does this change require those go.mod/sum modifications?

Reply:
Ya, I had a rogue file: core/services/chainlink/a_chainlink-packr.go

Removed now